.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -1472,13 +1471,19 @@ public class ChatMessage : IJsonModel<ChatMessage>, IPersistableModel<ChatMessag
         public static UserChatMessage CreateUserMessage(params ChatMessageContentPart[] contentParts);
         public static UserChatMessage CreateUserMessage(IEnumerable<ChatMessageContentPart> contentParts);
         public static UserChatMessage CreateUserMessage(string content);
-        public static implicit operator ChatMessage(string userMessage);
+        public static implicit operator ChatMessage(string content);
         ChatMessage IJsonModel<ChatMessage>.Create(ref Utf8JsonReader reader, ModelReaderWriterOptions options);
         void IJsonModel<ChatMessage>.Write(Utf8JsonWriter writer, ModelReaderWriterOptions options);
         ChatMessage IPersistableModel<ChatMessage>.Create(BinaryData data, ModelReaderWriterOptions options);
         string IPersistableModel<ChatMessage>.GetFormatFromOptions(ModelReaderWriterOptions options);
         BinaryData IPersistableModel<ChatMessage>.Write(ModelReaderWriterOptions options);
     }
+    public class ChatMessageContent : ObjectModel.Collection<ChatMessageContentPart> {
```

Comment:
`ChatMessageContent`'s representation in JSON is a JSON array rather than a JSON object. I assume MRW should read/write it as an array, but for now I simply chose to remove the `IJsonModel` interface. We can add it back later. In any case, the more common scenario would be to serialize the whole `ChatCompletion` or the whole `ChatMessage`, rather than just the content.